api.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+function fetchAllArticles(){
+   return axios.
+    get(`https://be-backend-project-nc-news.onrender.com/api/articles`).then((response)=>{
```

Comment:
well done for extracting this out - get it to directly return the articles data though as this is what the name suggests it does!